operation/fhir-operation-everything/src/main/java/com/ibm/fhir/operation/everything/EverythingOperation.java

```diff
@@ -176,6 +184,20 @@ protected Parameters doInvoke(FHIROperationContext operationContext, Class<? ext
         int maxPageSize = Math.max(1, FHIRConfigHelper.getIntProperty("fhirServer/core/maxPageSize", FHIRConstants.FHIR_PAGE_SIZE_DEFAULT_MAX));
         List<Entry> allEntries = new ArrayList<>(maxPageSize);
         allEntries.add(patientEntry);
+        List<String> resourceIds = new ArrayList<String>();
+        // Look up which extra resources should be returned
+        List<String> extraResources = FHIRConfigHelper.getStringListProperty("fhirServer/operations/everything");
+        // Patient is not part of its own compartment at this time.  If that behavior changes, remove these next lines
+        List<Entry> entryList = new ArrayList<Entry>(1);
+        entryList.add(patientEntry);
+        try {
+            readsOfAdditionalAssociatedResources(PATIENT, entryList, allEntries, resourceIds, resourceHelper, extraResources);
+        } catch (Exception e) {
+            FHIROperationException exceptionWithIssue = buildExceptionWithIssue("Error retrieving $everything "
+                    + "resources of type '" + PATIENT + "' for patient " + logicalId, IssueType.EXCEPTION, e);
```

Comment:
is this right?
I thought `readsOfAdditionalAssociatedResources`  did the reads of the `extraResources` that might be linked from the "newEntries" being passed.  but this message implies its actually reading the Patient resource instead.

Reply:
Changed the sentence in the catch clause to make it clearer.